types/storybook-addon-jsx/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "dependencies": {
+        "@storybook/addons": "^5.2.0",
+        "@storybook/react": "^5.2.0"
```

Comment:
Should `@storybook/react` be added to `devDependencies` since it's only used by the tests? 🤔

Reply:
CC @jameslnewell @AlexLomm @lonyele @mkornblum @fyrkant @RunningCoderLee @bradleyayers @Yama-Tomo @joscha @wapgear @dandean @ExE-Boss